profile_bluesky/startup/10-devices.py

```diff
@@ -12,10 +12,10 @@ class CompoundRefractiveLensDevice(Device):
 
 
 class MonochromatorDevice(Device):
-    monoE = Component(EpicsMotor, '8idimono:sm2', labels=["motor", "mono", "optics"])
-    monoth = Component(EpicsMotor, '8idimono:sm1', labels=["motor", "mono", "optics"])
+    Energy = Component(EpicsMotor, '8idimono:sm2', labels=["motor", "mono", "optics"])
```

Comment:
`energy`, not `Energy`

Reply:
@prjemian It has been fixed. Shall I generate a new Merge Pull Request?